src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -483,4 +485,50 @@ object Helpers extends SparkSessionWrapper {
     })
   }
 
+  /**
+   * Simplifies the acquisition of the workspace
+   * Requires that the ETLDB exists and has had successful previous runs
+   * As of 0.6.0.4
+   * Cannot derive schemas < 0.6.0.3
+   * @param etlDB Overwatch ETL database
+   * @return
+   */
+  def getWorkspaceByDatabase(etlDB: String): Workspace = {
+    // verify database exists
+    assert(spark.catalog.databaseExists(etlDB), s"The database provided, $etlDB, does not exist.")
+    val dbMeta = spark.sessionState.catalog.getDatabaseMetadata(etlDB)
+    val dbProperties = dbMeta.properties
+
+    // verify database is owned and managed by Overwatch
+    assert(dbProperties.getOrElse("OVERWATCHDB", "FALSE") == "TRUE", s"The database provided, $etlDB, is not an Overwatch managed Database. Please provide an Overwatch managed database")
+    val workspaceID = if (dbutils.notebook.getContext.tags("orgId") == "0") {
+      dbutils.notebook.getContext.tags("browserHostName").split("\\.")(0)
+    } else dbutils.notebook.getContext.tags("orgId")
```

Comment:
reuse function `getOrgId` defined earlier?

Reply:
good one, thx, added it to next PR